packages/nodes-base/nodes/Zammad/descriptions/TicketDescription.ts

```diff
@@ -0,0 +1,588 @@
+						},
+					},
+					{
+						displayName: 'Visibility',
+						name: 'visibility',
+						type: 'options',
+						default: 'internal',
+						options: [
+							{
+								name: 'External',
+								value: 'external',
+								description: 'Visible to customers',
+							},
+							{
+								name: 'Internal',
+								value: 'internal',
+								description: 'Visible to help desk',
+							},
+						],
+					},
+					{
+						displayName: 'Article Type',
+						name: 'type',
+						type: 'options',
+						// https://docs.zammad.org/en/latest/api/ticket/articles.html
+						options: [
+							{
+								name: 'Chat',
+								value: 'chat',
+							},
+							{
+								name: 'Email',
+								value: 'email',
+							},
+							{
+								name: 'Fax',
+								value: 'fax',
+							},
+							{
+								name: 'Note',
+								value: 'note',
+							},
+							{
+								name: 'Phone',
+								value: 'phone',
+							},
+							{
+								name: 'SMS',
+								value: 'sms',
+							},
+						],
+						default: 'note',
+					},
+				],
+			},
+		],
+	},
+	{
+		displayName: 'Additional Fields',
+		name: 'additionalFields',
+		type: 'collection',
+		displayOptions: {
+			show: {
+				resource: [
+					'ticket',
+				],
+				operation: [
+					'create',
+				],
+			},
+		},
+		default: {},
+		placeholder: 'Add Field',
+		options: [
+			{
+				displayName: 'Custom Fields',
+				name: 'customFieldsUi',
+				type: 'fixedCollection',
+				default: '',
+				placeholder: 'Add Custom Field',
+				typeOptions: {
+					multipleValues: true,
+				},
+				options: [
+					{
+						name: 'customFieldPairs',
+						displayName: 'Custom Field',
+						values: [
+							{
+								displayName: 'Field',
+								name: 'name',
+								type: 'options',
+								typeOptions: {
+									loadOptionsMethod: 'loadTicketCustomFields',
+								},
+								default: '',
+								description: 'Name of the custom field to set',
+							},
+							{
+								displayName: 'Value',
+								name: 'value',
+								type: 'string',
+								default: '',
+								description: 'Value to set on the custom field',
+							},
+						],
+					},
+				],
+			},
+		],
+	},
+	{
+		displayName: 'Update Fields',
+		name: 'updateFields',
+		type: 'collection',
+		displayOptions: {
+			show: {
+				resource: [
+					'ticket',
+				],
+				operation: [
+					'update',
+				],
+			},
+		},
+		default: {},
+		placeholder: 'Add Field',
+		options: [
+			{
+				displayName: 'Title',
+				name: 'title',
+				type: 'string',
+				default: '',
+			},
+			{
+				displayName: 'Group Name/ID',
+				name: 'group',
+				type: 'options',
+				typeOptions: {
+					loadOptionsMethod: 'loadGroupNames',
+				},
+				placeholder: 'First-Level Helpdesk',
+				description: 'Group that will own the ticket to create. Choose from the list or specify an ID using an <a href="https://docs.n8n.io/nodes/expressions.html#expressions">expression</a>.',
+				default: '',
+			},
+			{
+				displayName: 'Customer Email',
```

Comment:
- Because the entities exist in a fetchable number. Unlike an overly large number like Elasticsearch docs.
- Because the user can easily pick the entity, for better UX.
- Because the guide does not forbid this.